Inspicio/Controllers/AccountController.cs

```diff
@@ -122,6 +122,12 @@ public IActionResult Register(string returnUrl = null)
             if (ModelState.IsValid)
             {
                 var user = new ApplicationUser { UserName = model.Email, Email = model.Email };
```

Comment:
If you go with the ["usual" C# style](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions), it'll make it consistent with any 3rd party libraries you use (and make it feel less odd for anyone joining in if they're used to C#).